corehq/apps/domain/views.py

```diff
@@ -2527,6 +2536,23 @@ def page_context(self):
             'areas': dict([(a["name"], a["sub_areas"]) for a in settings.INTERNAL_DATA["area"]]),
         }
 
+    def send_handoff_email(self):
+        partner_contact = self.internal_settings_form.cleaned_data['partner_contact']
+        dimagi_contact = self.internal_settings_form.cleaned_data['dimagi_contact']
+        recipients = filter(None, [partner_contact, dimagi_contact])
```

Comment:
Why is the filter necessary here? It looks like they are both required fields (see form `clean()` method).